apps/remix-ide/src/app/tabs/locales/en/filePanel.json

```diff
@@ -39,10 +39,11 @@
   "filePanel.paste": "Paste",
   "filePanel.compile": "Compile",
   "filePanel.compileForNahmii": "Compile for Nahmii",
-  "filePanel.createNewFile": "Create New File",
-  "filePanel.createNewFolder": "Create New Folder",
-  "filePanel.publishToGist": "Publish all the current workspace files to a github gist",
-  "filePanel.uploadFile": "Load a local file into current workspace",
+  "filePanel.createNewFile": "Create new file",
+  "filePanel.createNewFolder": "Create new folder",
+  "filePanel.publishToGist": "Publish current workspace to GitHub gist",
+  "filePanel.uploadFile": "Upload files into current workspace",
+  "filePanel.uploadFolder": "Upload folder into current workspace",
```

Comment:
There is an inconsistency here. I don't think we need "current workspace" here.

Reply:
I agree

Reply:
updated